README.md

```diff
@@ -54,38 +54,37 @@ stages:
 unit2.7:
   image: python:2.7
   before_script:
-    - pip install -r requirements-dev.txt
-    - pip install -r requirements.txt
+	- pip install -r requirements-dev.txt
```

Comment:
These look over indented now.

Reply:
sorry, looks like my bad...  I will fix tomorrow

Reply:
Thanks.

Reply:
I think I restored the spacing Mike (sorry)